client/src/Components/HomePage/HomeNewsScreen.tsx

```diff
@@ -0,0 +1,105 @@
+import { ScrollView, TouchableOpacity } from 'react-native'
+import  React , {useEffect}  from 'react'
+import { NavigationProp, useNavigation } from '@react-navigation/native'
+import { RootStackParamList } from '../../types/navigation'
+import NewsForm from './NewsForm/NewsForm';
+import { observer } from 'mobx-react-lite'
+import Navigation from '../Navigation/Navigation'
+import axios from 'axios'
+import { NewsItem } from '../../types/NewsItem'
+import getNewsStore from '../../stores/newsStore'
+import CustomText from '../../CustomComponents/CustomText'
+import CustomView from '../../CustomComponents/CustomView'
+import getThemeStore from '../../stores/themeStore';
+import { darkMode, lightMode } from '../colors/colors';
+import { Dimensions } from 'react-native';
+
+const HomeNewsScreen : React.FC = observer(() => {
+    const navigation = useNavigation<NavigationProp<RootStackParamList>>();
+
+    const newsInformations = async() : Promise<void> => {
+        const IP_ADDRESS : string = "192.168.100.126"
```

Comment:
create a constant file for the ip and every constant u might use.

Reply:
DONE ✅